client/src/components/browse-recipes/Browse.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { onMount } from 'svelte';
+  import { slide } from 'svelte/transition';
+  import Header from '../header/Header.svelte';
+  import Navigation from '../navigation/Navigation.svelte';
```

Comment:
lets go for a global import here like so `@/components/navigation/Navigation.svelte`, `./` is acceptable but lets avoid `../`